melodic/distribution.yaml

```diff
@@ -9686,6 +9686,21 @@ repositories:
       url: https://github.com/KumarRobotics/ublox.git
       version: master
     status: maintained
+  ubt_msgs:
+    doc:
+      type: git
+      url: https://github.com/ubtvisbot/ubt_msgs.git
+      version: master
+    release:
+      tags:
+        release: release/melodic/{package}/{version}
+      url: https://github.com/ubtvisbot/ubt_msgs-release.git
+      version: 0.0.1-3
+    source:
+      type: git
+      url: https://github.com/ubtvisbot/ubt_msgs.git
```

Comment:
There is no license on this repository; the package.xml just says `TODO`, and there is no top-level LICENSE file in the repository.  Please add a license to this repository.

Reply:
Yes.  Otherwise, there is no way for a consumer of the package to tell whether they are allowed to use the package in their application.  For instance, some companies can't use proprietary licensed software.  Some can't use GPL-licensed software.  Without this information, they can't make a determination on whether they can use your software.